src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.5.0-beta.1
```

Comment:
I would put here 1.0.0-beta.5.

Lets wait for other @open-telemetry/dotnet-contrib-maintainers

Reply:
Thanks for quick review. It was not entirely clear what the convention is (first contrib here), so my reasoning was to follow the OTEL v1.5 version as I saw this in some other package releases like `OpenTelemetry.Instrumentation.Http` and `OpenTelemetry.Instrumentation.AspNetCore` about 2 weeks ago.

I'll wait for further feedback before updating!

Reply:
I did make an oversight in the date though :( Jul->Jun

Reply:
Both cases should work, last big chunk of releases is based on just rising -beta.x suffix: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1223

Reply:
I always prefer follow a convention :)
For me both are good but I have a preference for a versioning decoupled of dependencies.
Because in this case, we have two dependencies.

Reply:
I think it should be fine to follow what the main repo instrumentation libraries are doing and use `1.5.0-beta.1` here.